diary/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+# from django.urls import reverse
+from rest_framework import generics
+from rest_framework.views import APIView
+from rest_framework import status
+from rest_framework.response import Response
+from .models import User
+from .serializers import UserSignupSerializer
+from .tokens import ACTIVATIONTOKEN
 
-# Create your views here.
+
+class UserSignUp(generics.CreateAPIView):
+    serializer_class = UserSignupSerializer
+    queryset = User.objects.all()
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        self.perform_create(serializer)
+        result = {
+            'user': serializer.data,
+            'message': "Successfully registered, please check your email to activate your account."
+        }
+        return Response(result, status=status.HTTP_201_CREATED)
+
+
+class ActivateAccount(APIView):
+    def get(self, request, slug_field=None):
+        token = request.GET.get('token')
+        try:
+            user = User.objects.get(slug_field=slug_field)
+            # print(ACTIVATIONTOKEN.check_token(user, token))
```

Comment:
I think this print statement should be removed.

Reply:
Thank you.